website/docs/r/app_service_github_token.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "App Service (Web Apps)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_github_token"
+description: |-
+  Manages an App Service GitHub Token.
+---
+
+# azurerm_app_service_github_token
+
+Manages an App Service GitHub Token.
+
+~> **NOTE:** This resource can only manage the token for the user currently running Terraform. Managing tokens for another user is not supported by the service. 
+
+
```

Comment:
```suggestion
```

Reply:
is this beta gated resource and should be called out/guide linked?

Reply:
I missed re-naming the file when I changed from specific token type to generic, I'll correct this.